### DISCUSSION TITLE
PPLNS payment

### DISCUSSION THREAD

**Discussion by szilardkoppel** [coinfoundry/miningcore#967] [Q&A] [answered]
Hello,

i want to ask that what factor means in payoutSchemeConfig. We successfully mined a block on main net but only 49.5% paid out.

`[PPLNS Payment] Balance-calculation for pool erg1, block 584055 completed with accumulated score 0.2473 (49.5%)`

I need to set it to 1 to pay out the full amount? currently on 0.5. what happens if I modify it, is affecting the next payment?

Thank you!

**Comment by oliverw** [comment, marked as answer]
No you don't need to set it to 1. The part of the message about the accumulated score basically means that the very first block the pool ever found was lucky and there weren't simply enough total shares (of all miners combined) to satisfy the calculation. There is a small chance for this to happen for the next one or two blocks as well if the pool is very lucky and after that it won't happen again. From here you have two options:

1. Treat the remaining block reward as a reward for the pool (you) because well, luck
2. Manually repeat previous payments to miners by altering the `balances` table, setting the `amount` column to the value of the previous/first payout

Explanation for the `factor` property (`factor` is `X`):  https://bitcointalk.org/index.php?topic=39832

**Reply by szilardkoppel** [reply]
Thank you very much for you fast reply and great explanation! If you accept a coffee please tell me your erg address :)

**Reply by oliverw** [reply]
> Thank you very much for you fast reply and great explanation! If you accept a coffee please tell me your erg address :)

You're welcome.
